Sources/Components/Consent/BannerTransparency/BannerTransparencySectionView.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+protocol BannerTransparencySectionViewDelegate: AnyObject {
+    func bannerTransparencySectionViewDidSelectExternalLinkButton(_ view: BannerTransparencySectionView)
+}
+
+final class BannerTransparencySectionView: UIView {
+    weak var delegate: BannerTransparencySectionViewDelegate?
+
+    private lazy var titleLabel: Label = {
+        let label = Label(style: .title3)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 0
+        return label
+    }()
+
+    private lazy var textLabel: UILabel = {
```

Comment:
Nitpicking, can this have a more descriptive name?

Reply:
Sure, can rename it to `descriptionTextLabel` for example.

Reply:
Actually I renamed it to `detailTextLabel` and also introduced one more view model for internal use.

Reply:
🚢 🚢